README.md

```diff
@@ -143,13 +143,32 @@ Campers will likely use Guide articles as a quick reference to look up syntax. A
 
 Here are specific formatting guidelines for any code:
 
-- Use two spaces to indent
 - JavaScript statements end with a semicolon
 - Use double quotes where applicable
 - Show generally-accepted best practices, particularly for accessibility
 - Comments made should have a space between the comment characters and the comment themselves
 
     `// Fix this line`
+- To use syntax highlighting in code blocks, indicate the language after starting ```
+```
+    ```html
+        <div class='awesome' id='more-awesome'>
+          <p>This is text in html</p>
+        </div>
+    ``` 
+
+    ```javascript
+        function logTheThings(stuff) {
+         console.log(stuff);
+        }
+    ``` 
+
+    ```css
+       .awesome {
+          background-color: #BADA55;
```

Comment:
These examples are great 😆 - I'd just suggest tweaking the CSS color to `#FCCFCC` - while most of fCC's audience are adults, we are mindful just in case any kiddos stumble over here 😃 . Otherwise, this looks good to merge 👍

Reply:
@HKuz Sure. Updating the css.